docs/users/wallets/ledger.md

```diff
@@ -79,6 +79,35 @@ Now you can import your Ledger account to MetaMask by using the following steps:
 
 ![mm3.png](./../../img/mm3.png)
 
+### Evmos CLI
+
+To use your Ledger with the Evmos CLI, first connect your device to your computer, unlock it using your PIN, and open the Ethereum app.
+
+Then, connect your Ledger to the CLI with the following command:
+
+`evmosd keys add myledger --ledger`
```

Comment:
nit, use Google CLI syntax for variables that the user can choose. Also use code blocks for commands

```
evmosd keys add NAME --ledger
```